src/main/java/com/hivemq/bootstrap/ClientConnection.java

```diff
@@ -375,7 +374,7 @@ public void setQueueSizeMaximum(final @Nullable Long queueSizeMaximum) {
         this.queueSizeMaximum = queueSizeMaximum;
     }
 
-    public @NotNull MessageIDPool getMessageIDPool() {
+    public @NotNull FreePacketIdRanges getMessageIDPool() {
         return messageIDPool;
```

Comment:
Maybe change the wording from message ID pool to free packet ID ranges everywhere.